src/types/config.ts

```diff
@@ -233,6 +233,10 @@ export interface DefaultDBConfig {
    * These will not be prefixed, so type the exact name.
    */
   shared?: string[];
+  /**
+   * Internal databases, that will be created by createMissingDBs() in `user.ts`
+   */
+  internal?: string[];
```

Comment:
Like this, the model is misleading. `private` and `shared` only make sense in the context of databases for users. I'd suggest to simply use the `SystemDBConfig` and have an array of system DBs there which will be created with the default permissons. 

In a future iteration, we could add functionality according to PersonalDBSettings/ PersonalDBModel to support assigning roles, creating DBs as partitioned, etc